Assets/MRTK/SDK/Experimental/Drawstring/Scripts/TwoHandleManipulator.cs

```diff
@@ -0,0 +1,937 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.Input;
+using Microsoft.MixedReality.Toolkit.Physics.Experimental;
+using Microsoft.MixedReality.Toolkit.Utilities;
+using Microsoft.MixedReality.Toolkit.Physics;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using TMPro;
+using UnityEngine;
+using UnityEngine.Assertions;
+using UnityEngine.Serialization;
+using UnityEngine.UI;
+
+namespace Microsoft.MixedReality.Toolkit.UI.Experimental
+{
+    /// <summary>
+    /// TODO insert drawstring summary
```

Comment:
Nit: Don't check in code with TODOs (or comments with TODOs). Either do the work now or file an issue you can link to to provide future people with background for what needs to be changed.